template.php

```diff
@@ -21,6 +21,22 @@
 include 'includes/theme_webform_managed_file.inc';
 include 'includes/theme_webform_time.inc';
 
+/**
+ * Make theme layouts available.
+ */
+function campaignion_foundation_campaignion_layout_info() {
```

Comment:
Isn't there a `Implements hook_campaignion_layout_info().` missing?

Reply:
Maybe. Maybe not. I feel like [Drupal docstings](https://www.drupal.org/docs/develop/standards/api-documentation-and-comment-standards) are science of their own that I have not mastered yet.

Reply:
Yep, proper style would be to have `Implements …` as the first line in the comment.

Reply:
Only for this hook? Or everything else in the file that is not a preprocess or custom function, too?

Reply:
@katzmo It’s the standard for all hook implementations.